### PR TITLE
Remove old PoC code branding

### PR DESCRIPTION
**Pull request by Diabelko**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -107,15 +107,15 @@
...
+htmlhelp_basename = 'ACID'
 
 
 # -- Options for manual page output ------------------------------------------
 
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
 man_pages = [
-    (master_doc, 'TungstenCIDashboard', 'Tungsten CI dashboard Documentation',
+    (master_doc, 'ACIDCIDashboard', 'ACID CI dashboard Documentation',
```
Comment (kawazaki0) [inline docs/conf.py]: The `ACIDCIDashboard` is an ugly creature. Maybe use only `ACID`?

Comment (pawelzny) [reply docs/conf.py]: And `ACID CI dashboard Documentation` also looks bad. Maybe just `ACID Documentation`.

File: playbooks/roles/run-zuul/templates/main.yaml.j2 [hunk 2]
```diff
@@ -1,5 +1,5 @@
 - tenant:
-    name: Tungsten Fabric
+    name: ACID
```
Comment (pawelzny) [inline playbooks/roles/run-zuul/templates/main.yaml.j2]: IMHO right now Codilime is a tenant, isn't it?

File: docs/conf.py [hunk 3]
```diff
@@ -21,9 +21,9 @@
 
 # -- Project information -----------------------------------------------------
 
-project = 'Tungsten CI Dashboard'
-copyright = u'2018, Tungsten Fabric'
-author = u'Tungsten Fabric'
+project = 'ACID CI Dashboard'
```
Comment (pawelzny) [inline docs/conf.py]: IMHO it should be `ACID` or `Awesome CI Dashboard` unless we are going to use recursive naming.

File: playbooks/roles/run-zuul/templates/main.yaml.j2 [hunk 4]
```diff
@@ -1,5 +1,5 @@
 - tenant:
-    name: Tungsten Fabric
+    name: CodiLime
```
Comment (pawelzny) [inline playbooks/roles/run-zuul/templates/main.yaml.j2]: Tego nie wolno bo gerrit przestanie działać. Poza tym jestem prawie pewien ze to zmieniłem na "acid".

**Pull request merged**